core/services/pipeline/task.bridge_test.go

```diff
@@ -353,6 +355,58 @@ func TestBridgeTask_DoesNotReturnStaleResults(t *testing.T) {
 
 	require.Error(t, result2.Error)
 	require.Nil(t, result2.Value)
+
+	task2 := pipeline.BridgeTask{
+		BaseTask:    pipeline.NewBaseTask(0, "bridge2", nil, nil, 0),
+		Name:        bridge.Name.String(),
+		RequestData: btcUSDPairing,
+		CacheTTL:    "35m", // more than the stalenessCap 30m
+	}
+	task2.HelperSetDependencies(cfg2, orm, specID, uuid.UUID{}, c)
+
+	// Insert entry 32m in the past, under cacheTTL of 35m but more than stalenessCap of 30m.
+	err = queryer.ExecQ(`INSERT INTO bridge_last_value(dot_id, spec_id, value, finished_at)
+		VALUES($1, $2, $3, $4) ON CONFLICT ON CONSTRAINT bridge_last_value_pkey
+		DO UPDATE SET value = $3, finished_at = $4;`, task2.DotID(), specID, big.NewInt(9700).Bytes(), time.Now().Add(-32*time.Minute))
+	require.NoError(t, err)
+
+	// Run fails even though cacheTTL > lastvalue.finished_at because cacheTTL exceeds stalenessCap.
+	result2, _ = task2.Run(testutils.Context(t), logger.TestLogger(t),
+		pipeline.NewVarsFrom(
+			map[string]interface{}{
+				"jobRun": map[string]interface{}{
+					"meta": map[string]interface{}{
```

Comment:
If you wanted to condense this a little bit, you could use `any`:
```suggestion

	// Run fails even though cacheTTL > lastvalue.finished_at because cacheTTL exceeds stalenessCap.
	result2, _ = task2.Run(testutils.Context(t), logger.TestLogger(t),
		pipeline.NewVarsFrom(
			map[string]any{
				"jobRun": map[string]any{
					"meta": map[string]any{
```
Or even a type alias like this might work: :thinking: 
```suggestion
	type Map = map[string]any
	// Run fails even though cacheTTL > lastvalue.finished_at because cacheTTL exceeds stalenessCap.
	result2, _ = task2.Run(testutils.Context(t), logger.TestLogger(t),
		pipeline.NewVarsFrom(
			Map{
				"jobRun": Map{
					"meta": Map{
```